.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+1. **What type of issue would you like to report**?
+
+- [ ] Bug
+- [ ] Feature request
+- [ ] Other, please explain:
+
+2. **Check if the problem you are trying to report is already reported**.
+
+Please look for the existing issues covering the same case. First look on the [GitHub issue page](https://github.com/ckeditor/ckeditor-dev/issues)
+and then, if nothing was found, look in [CKEditor former bug reporting system](http://dev.ckeditor.com).
+
+If there is already an issue reported on [GitHub issue page](https://github.com/ckeditor/ckeditor-dev/issues) there is no
+need to create a duplicate. Just add your reaction (+1) there to upvote the issue. If you have any additional comments you
+may also add it there.
+
+If you found any similar issues in [CKEditor former bug reporting system](http://dev.ckeditor.com),
+please add all references (links) below:
+
+3. **If you are reporting a bug, provide a reproduction steps**.
+
+Steps to reproduce
+
+1. ...
+2. ...
+3. ...
+
+Expected result
```

Comment:
Should be a header, same with actual results, other details. As for "Steps to reproduce" ideally we'd like to somehow join it with "If you are reporting a bug, provide a reproduction steps"